src/pdfkit.cs

```diff
@@ -917,69 +1870,85 @@ interface PdfView : NSMenuDelegate {
 		CGSize RowSize (PdfPage page);
 
+		[NoiOS]
 		[Export ("allowsDragging")]
 		bool AllowsDragging { get; set; }
 	
-		//Verify NSArray
 		[Export ("visiblePages")]
 		PdfPage [] VisiblePages { get; }
 
 		[Export ("enableDataDetectors")]
 		bool EnableDataDetectors { get; set; }
 
-		[Field("PDFViewChangedHistoryNotification")]
+		[Field("PDFViewChangedHistoryNotification", "+PDFKit")]
 		[Notification]
 		NSString ChangedHistoryNotification { get; }
 
-		[Field("PDFViewDocumentChangedNotification")]
+		[Field("PDFViewDocumentChangedNotification", "+PDFKit")]
 		[Notification]
 		NSString DocumentChangedNotification { get; }
 
-		[Field ("PDFViewPageChangedNotification")]
+		[Field ("PDFViewPageChangedNotification", "+PDFKit")]
 		[Notification]
 		NSString PageChangedNotification { get; }
 
-		[Field ("PDFViewScaleChangedNotification")]
+		[Field ("PDFViewScaleChangedNotification", "+PDFKit")]
 		[Notification]
 		NSString ScaleChangedNotification { get; }
 
-		[Field ("PDFViewAnnotationHitNotification")]
+		[Field ("PDFViewAnnotationHitNotification", "+PDFKit")]
 		[Notification (typeof (PdfViewAnnotationHitEventArgs))]
 		NSString AnnotationHitNotification { get; }
 
-		[Field ("PDFViewCopyPermissionNotification")]
+		[Field ("PDFViewCopyPermissionNotification", "+PDFKit")]
 		[Notification]
 		NSString CopyPermissionNotification { get; }
 
-		[Field ("PDFViewAnnotationWillHitNotification")]
+		[Field ("PDFViewPrintPermissionNotification", "+PDFKit")]
+		[Notification]
+		NSString PrintPermissionNotification { get; }
+
+		[Field ("PDFViewAnnotationWillHitNotification", "+PDFKit")]
 		[Notification]
 		NSString AnnotationWillHitNotification { get; }
 
-		[Field ("PDFViewSelectionChangedNotification")]
+		[Field ("PDFViewSelectionChangedNotification", "+PDFKit")]
 		[Notification]
 		NSString SelectionChangedNotification { get; }
 
-		[Field ("PDFViewDisplayModeChangedNotification")]
+		[Field ("PDFViewDisplayModeChangedNotification", "+PDFKit")]
 		[Notification]
 		NSString DisplayModeChangedNotification { get; }
 
-		[Field ("PDFViewDisplayBoxChangedNotification")]
+		[Field ("PDFViewDisplayBoxChangedNotification", "+PDFKit")]
 		[Notification]
 		NSString DisplayBoxChangedNotification { get; }
+
+		[Field ("PDFViewVisiblePagesChangedNotification", "+PDFKit")]
+		[Notification]
+		NSString VisiblePagesChangedNotification { get; }
+
+		//[NoiOS]
+		//[NoMac] // [Mac (10,13)] Headers do not show changes on Mac
+		//[Export ("acceptsDraggedFiles")]
+		//bool AcceptsDraggedFiles { get; set; }
 	}
 	
+	[NoiOS]
 	interface PdfViewAnnotationHitEventArgs {
 		[Export ("PDFAnnotationHit")]
 		PdfAnnotation AnnotationHit { get; }
 	}
 	
 	//Verify delegate methods.  There are default actions (not just return null ) that should occur
 	//if the delegate does not implement the method.
-	[BaseType (typeof (NSObject))]
+	[iOS (11,0)]
+	[BaseType (typeof (NSObject), Name = "PDFViewDelegate")]
 	[Model]
-	[Protocol (IsInformal = true)]
+	[Protocol]
```

Comment:
It's still wrong, since it's formal _now_ :smile:

There's a FormalSince field that should be added when an informal protocol turns formal:

```
[Protocol (FormalSince = "10.12")]
```